README.md

```diff
@@ -25,15 +25,16 @@ $ cd onlineweb4
 
 To fire up the dev environment, you should use docker-compose.
 
-Install it by running `pip install docker-compose`.
+First install Docker and Docker-compose 
```

Comment:
Why did you remove one of the install instructions? I understand that not everyone wants to use `pip`, but since we use Python most people probably have it installed.